### PR TITLE
node-telegram-bot-api: support sendPoll, stopPoll, and poll_answer event

### PR DESCRIPTION
**Pull request by XieJiSS**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://core.telegram.org/bots/api

### CODE REVIEW

File: types/node-telegram-bot-api/index.d.ts [hunk 1]
```diff
@@ -647,6 +674,8 @@ declare namespace TelegramBot {
         set_name?: string;
         mask_position?: MaskPosition;
         file_size?: number;
+        file_unique_id?: string;
```
Comment (XC-Zhang) [inline types/node-telegram-bot-api/index.d.ts:677]: Make `file_unique_id` required? It is not optional in the [docs](https://core.telegram.org/bots/api#sticker).

File: types/node-telegram-bot-api/index.d.ts [hunk 2]
```diff
@@ -647,6 +674,8 @@ declare namespace TelegramBot {
...
+        is_animated?: boolean;
```
Comment (XC-Zhang) [inline types/node-telegram-bot-api/index.d.ts:678]: Make `is_animated` required? It is not optional in the [docs](https://core.telegram.org/bots/api#sticker).

**Review by XC-Zhang — CHANGES_REQUESTED** [review]
Thanks for PR! Please see comments I made.

By the way, since poll_answer event has been added to `on`, I suggest that it should also be added to `addListener`.

There is one more thing I'd like to discuss. Poll type currently can be "regular" or "quiz". Should we use [String Literal Types](https://www.typescriptlang.org/docs/handbook/advanced-types.html#string-literal-types) instead of `string`? For example:
```typescript
type PollType = "regular" | "quiz";
interface Poll {
    // ...
    type: PollType;
    // ...
}
```

**Review by XC-Zhang — APPROVED** [review]

**Comment by XieJiSS** [comment]
Sorry, but when I was re-checking these codes, I realized that I missed a test for `addListener` 😢

**Review by XC-Zhang — APPROVED** [review]

**Comment by XC-Zhang** [comment]
> Sorry, but when I was re-checking these codes, I realized that I missed a test for `addListener` 😢

Good point! 👍

**Comment by XC-Zhang** [comment]
Although not that important, interface `SendPollOptions` could be improved by using `PollType` you added yesterday:

**Comment by XieJiSS** [comment]
@XC-Zhang Nice point! I've pushed another commit to fix that.

**Review by XC-Zhang — APPROVED** [review]
Excellent job! Let's merge it! 🎉

**Comment by XieJiSS** [comment]
Ready to merge

**Pull request merged**